pipelines/nextflow/subworkflows/run_busco.nf

```diff
@@ -44,49 +42,83 @@ include { BUSCO_CORE_METAKEYS as BUSCO_CORE_METAKEYS_GENOME } from '../modules/b
-        //if (copyToFtp) {
-        //    COPY_PROTEIN_OUTPUT(buscoProteinSummaryOutput)
-        //}
-        //def buscoProteinSummaryOutput1=buscoProteinSummaryOutput
-        if(params.apply_busco_metakeys){
-            BUSCO_CORE_METAKEYS_PROTEIN(buscoProteinSummaryOutput)
+        // Get the closest BUSCO dataset from the taxonomy classification stored in db meta table
+        BUSCO_DATASET(db_meta).clade_dataset
+            .map { row -> [
+                insdc_acc: row[0], taxonomy_id: row[1], core: row[2], production_name: row[3],
+                organism_name: row[4], annotation_source: row[5], ortho_db: row[6]
+            ]}
+            .set{ orthodb_amended_meta }
+
+        // Run BUSCO in genome mode
+        if (busco_mode.contains('genome')) {
+
+            output_typeG = "genome"
+
+            // Download genome via ncbi dataset API
+            FETCH_GENOME (orthodb_amended_meta).genome_fasta
+                .map { row -> [
+                    insdc_acc: row[0], taxonomy_id: row[1], core: row[2], production_name: row[3],
+                    organism_name: row[4], annotation_source: row[5], ortho_db: row[6], genome: row[7]
+                ]}
+                .set{ metadata_genome_fna }
+
+            BUSCO_GENOME_LINEAGE(metadata_genome_fna, output_typeG).busco_report_output
+                .map { row -> [
+                    insdc_acc: row[0], taxonomy_id: row[1], core: row[2],production_name: row[3],
+                    organism_name: row[4], annotation_source: row[5], report: row[6]
+                ]}
+                .set{ buscoGenomeOutput }
+
+            buscoGenomeSummaryOutput = BUSCO_GENOME_OUTPUT(buscoGenomeOutput, output_typeG)
+
+            // Copy BUSCO summary stats to ensembl FTP
+            // if (copyToFtp) {
+            //     COPY_GENOME_OUTPUT(buscoGenomeSummaryOutput)
+            // }
+
+            // Make and apply busco summary meta_keys patch directly to core:
+            if(params.apply_busco_metakeys){
+                BUSCO_CORE_METAKEYS_GENOME(buscoGenomeSummaryOutput)
+            }
         }
 
-    }
-}
+        // Run Busco in protein mode
+        if (busco_mode.contains('protein')) {
+
+            output_typeP = "protein"
 
+            // Dump protein translations
+            FETCH_PROTEINS (orthodb_amended_meta).translations
+                .map { row -> [
+                    insdc_acc: row[0], taxonomy_id: row[1], core: row[2], production_name: row[3],
+                    organism_name: row[4], annotation_source: row[5], ortho_db: row[6], translations: row[7]
+                ]}
+                .set{ metadata_prot_trans }
 
+            BUSCO_PROTEIN_LINEAGE(metadata_prot_trans, output_typeP).busco_report_output
+                .map { row -> [
+                    insdc_acc: row[0], taxonomy_id: row[1], core: row[2], production_name: row[3],
+                    organism_name: row[4], annotation_source: row[5], report: row[6]
```

Comment:
what is the report variable?

Reply:
Just the `busco_report_output` summary report generated within the BUSCO_LINEAGES module